devops/containers/ubuntu2204_intel_drivers.Dockerfile

```diff
@@ -6,8 +6,9 @@ FROM $base_image:$base_tag
 ENV DEBIAN_FRONTEND=noninteractive
 
 ARG use_latest=true
+ARG use_igc_dev=false
 
-RUN apt update && apt install -yqq wget
+RUN apt update && apt install -yqq wget libllvm14
```

Comment:
I'm not sure we want LLVM installed in the image unconditionally.

Reply:
yeah, thought about that too, but this is lib only,  it is not harmful to have it. :)
But sure, let us avoid polluting existing images.
Done.